libensemble/tests/regression_tests/test_ytopt_heffte.py

```diff
@@ -0,0 +1,137 @@
+"""
+Runs libEnsemble to call the ytopt ask/tell interface in a generator function,
+and the ytopt findRunTime interface in a simulator function.
+
+Execute locally via one of the following commands (e.g. 3 workers):
+   mpiexec -np 4 python run_ytopt_xsbench.py
+   python run_ytopt_xsbench.py --nworkers 3 --comms local
+
+The number of concurrent evaluations of the objective function will be 4-1=3.
+"""
+
```

Comment:
missing TESTSUITE_ fields?

Reply:
Yes. I don't think we are (or should) build heffte in all cases. I've added them.